Sources/Utility/ArgumentParser.swift

```diff
@@ -225,15 +263,33 @@ public final class OptionArgument<Kind>: ArgumentProtocol {
     typealias ArgumentKindTy = Kind
 
     let name: String
-    let usage: String?
     let shortName: String?
+    // Option arguments are always optional
+    var isOptional: Bool { return true }
+    let strategy: ArrayParsingStrategy
+    let usage: String?
 
-    init(name: String, shortName: String?, usage: String?) {
+    init(name: String, shortName: String?, strategy: ArrayParsingStrategy, usage: String?) {
         precondition(!isPositional(argument: name))
         self.name = name
         self.shortName = shortName
+        self.strategy = strategy
         self.usage = usage
     }
+
+    func parse(_ kind: ArgumentKind.Type, with parser: inout ArgumentParserProtocol) throws -> [ArgumentKind] {
+        if let associatedArgument = parser.associatedArgumentValue {
```

Comment:
We should write a comment here that if we encounter associated value, we ignore the strategy.